pandas/tests/scalar/period/test_period.py

```diff
@@ -73,10 +73,8 @@ def test_construction(self):
 
         i1 = Period.now(freq="Q")
         i2 = Period(datetime.now(), freq="Q")
-        i3 = Period.now("q")
```

Comment:
could you remind me why this is removed please?

Reply:
as is it just because we're not uppercasing qe? if so, sounds fine tbh

Reply:
we had a strange behaviour, the same ass for deprecation M to ME:

```
Period('2020-01', 'QE') => errors
Period('2020-01', 'qe') => silently converts to 'M'
Period('2020-01', 'q') => errors
```
I added `'qe' `to `_dont_uppercase` and it fixed the problem. I left a comment linking to #54105, saying that this should raise in the future an error.